### PR TITLE
Add lat and lon columns for `[adiff ... out * center;` queries in `osmdata_data_frame`

### PR DESCRIPTION
**Pull request by jmaspons**

### CODE REVIEW

File: codemeta.json [hunk 1]
```diff
@@ -4,20 +4,17 @@
...
-  "relatedLink": [
-    "https://docs.ropensci.org/osmdata/",
-    "https://CRAN.R-project.org/package=osmdata"
-  ],
+  "relatedLink": ["https://docs.ropensci.org/osmdata/", "https://CRAN.R-project.org/package=osmdata"],
   "codeRepository": "https://github.com/ropensci/osmdata/",
   "issueTracker": "https://github.com/ropensci/osmdata/issues",
   "license": "https://spdx.org/licenses/GPL-3.0",
-  "version": "0.2.1.002",
+  "version": "0.2.1.3",
```
Comment (jmaspons) [inline codemeta.json:11]:
```suggestion
  "version": "0.2.1.003",
```

File: codemeta.json [hunk 2]
```diff
@@ -4,20 +4,17 @@
...
   "programmingLanguage": {
     "@type": "ComputerLanguage",
     "name": "R",
     "url": "https://r-project.org"
   },
-  "runtimePlatform": "R version 4.2.2 (2022-10-31)",
+  "runtimePlatform": "R version 4.2.2 Patched (2022-11-10 r83330)",
```
Comment (jmaspons) [inline codemeta.json:17]:
```suggestion
  "runtimePlatform": "R version 4.2.2 Patched (2022-10-31)",
```

File: codemeta.json [hunk 3]
```diff
@@ -348,25 +345,12 @@
...
-    "cpp",
-    "rstats",
-    "osm",
-    "osm-data",
-    "r-package",
-    "peer-reviewed"
-  ],
-  "fileSize": "14961.933KB",
+  "keywords": ["open0street0map", "openstreetmap", "overpass0API", "OSM"],
+  "fileSize": "45067.501KB",
```
Comment (jmaspons) [inline codemeta.json:353]:
```suggestion
  "fileSize": "14961.933KB",
```

**Review by mpadge — APPROVED** [review]
That all looks great. I make the comment somewhere else that it would be good someday to translate this code into C++, but since:

1. all loops here are single-level, via `vapply`/`lapply`, and
2. adiff queries can generally be presumed to be much smaller than "standard" queries

then I think it's still okay for the moment to leave as is. Given which: this is great, thank you!

**Pull request merged**